DIPs/13NNN-WGB.md

```diff
@@ -428,12 +428,92 @@ Class objects cannot have move constructors or move assignment operators.
 ### Last Use
 
 The *Last Use* of an lvalue is the last time in a path of execution that any read
-or write is done to the memory region referred to directly by the lvalue. For example:
+or write is done to the memory region referred to directly by the lvalue.
 
+**Return Statements**. If an lvalue is returned directly via `return x`, that is the last use of `x`.
```

Comment:
what if `x` is a global?

Reply:
There's never a last use of global. Only locals and parameters are checked.